client/WEB-INF/classes/resources/messages_es.properties

```diff
@@ -808,7 +808,7 @@ label.skip.guide=He utilizado CloudStack anteriormente, saltar esta gu\u00eda
 label.snapshot=Instant\u00c3\u00a1nea
 label.snapshot.limits=instant\u00c3\u00a1neas L\u00c3\u00admites
 label.snapshot.name=Nombre de instant\u00c3\u00a1neas
-label.snapshot.schedule=Lista de instant\u00c3\u00a1neas
+label.snapshot.s=instant\u00c3\u00a1neas
```

Comment:
you are removing label.snapshot.schedule as well, that was not changed in the commit from Mike, is that intended?